src/Microsoft.SqlTools.ServiceLayer/SqlProjects/Contracts/SqlCmdVariables/AddSqlCmdVariable.cs

```diff
@@ -24,11 +24,6 @@ public class AddSqlCmdVariableParams : SqlProjectParams
         /// Default value of the SQLCMD variable
         /// </summary>
         public string DefaultValue { get; set; }
-
-        /// <summary>
-        /// Value of the SQLCMD variable, with or without the $()
-        /// </summary>
-        public string Value { get; set; }
```

Comment:
Be sure to also remove this from the vscode-mssql extension!

Reply:
yup will do! Thanks for the reminder!